contracts/bepro/Network_v2.sol

```diff
@@ -437,32 +391,26 @@ contract Network_v2 is Governed, ReentrancyGuard {
 
     /// @dev enable users to fund a bounty
     function fundBounty(uint256 id, uint256 fundingAmount) external payable {
-        // _bountyExists(id);
+
         _isFundingRequest(id, true);
         _isInDraft(id, true);
         _isNotCanceled(id);
 
         Bounty storage bounty = bounties[id];
         require(bounty.funded == false, "F1");
         require(bounty.tokenAmount < bounty.fundingAmount, "F2");
-        require(bounty.tokenAmount < bounty.tokenAmount.add(fundingAmount), "F3");
```

Comment:
you are allowing to someone to add funds more than specified on the bounty. Is it on purpose ?

Reply:
Nope. foresight!

Reply:
Added it back in with the `=` logic we spoke of